### PR TITLE
Fixing some bugs and starting to add tests

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: src/utils/PolicyAdapter.ts [hunk 1]
```diff
@@ -13,36 +13,39 @@ import { DeepPartial } from 'ts-essentials';
...
                 return '';
             }
 
-            let encodedAction = `${action.type} `;
+            let encodedAction = `${action.type}`;
 
             switch (action.type) {
                 case ActionType.WEBHOOK:
-                    encodedAction += action.endpoint;
+                    encodedAction += ' ' + action.endpoint;
```
Comment (pilhuhn) [inline src/utils/PolicyAdapter.ts]: Are you sure that email is always 1st and webhook 2nd? Can't this be the other way around?

Comment (josejulio) [reply src/utils/PolicyAdapter.ts]: I don't follow. Why are you asking if email is always 1st? Where is that implied?

Comment (josejulio) [reply src/utils/PolicyAdapter.ts]: You mean the tests? They are processed in the order that the array has.

Comment (pilhuhn) [reply src/utils/PolicyAdapter.ts]: I misread the code. You add the opt-arg for hooks after the space. And I read that you iterate over actions and add the action name.

**Pull request merged**